### ISSUE TITLE
npm failing to install gulp with error: {code: EPERM, errno: -4048, syscall: rename }

### ISSUE THREAD

**Issue by bobheath33435** [npm/npm#16508]
#### I'm opening this issue because:

  - [x ] npm is crashing.
  - [x ] npm is producing an incorrect install.
  - [x ] npm is doing something I don't understand.
  - [ ] Other (_see below for feature requests_):

### supporting information:

 - `npm -v` prints: 3.10.9
 - `node -v` prints: v6.9.1
 - `npm config get registry` prints:
 - Windows, OS X/macOS, or Linux?:
 - Network issues: https://registry.npmjs.org/
   - Geographic location where npm was run:
   - [ ] I use a proxy to connect to the npm registry.
   - [x ] I use a proxy to connect to the web.
   - [ ] I use a proxy when downloading Git repos.
   - [ ] I access the npm registry via a VPN
   - [ ] I don't use a proxy, but have limited or unreliable internet access.
 - Container:
   - [ ] I develop using Vagrant on Windows.
   - [ ] I develop using Vagrant on OS X or Linux.
   - [ ] I develop / deploy using Docker.
   - [ ] I deploy to a PaaS (Triton, Heroku).

**Comment by bobheath33435** [comment]
I have found a 'work-around'. It seems that I could not perform the npm install command while the directory was in the parent directory tree. So I took that directory out of the parent's directory tree to perform all of the npm installs for gulp, which worked fine. Once I had all of the necessary gulp modules installed, I just moved the directory back to its original place in the parent's directory tree. Gulp worked fine.
